src/jailer/src/env.rs

```diff
@@ -159,6 +159,22 @@ impl Env {
 
         // Optional arguments.
         let mut cgroups: Vec<Box<dyn Cgroup>> = Vec::new();
+        let parent_cgroup = match arguments.single_value("parent-cgroup") {
+            Some(parent_cg) => Path::new(parent_cg),
+            None => Path::new(exec_file_name),
+        };
+        if parent_cgroup.is_absolute() {
+            return Err(Error::CgroupInvalidParentPath());
+        }
+        for c in parent_cgroup.components() {
+            match c {
+                Component::CurDir | Component::ParentDir => {
+                    return Err(Error::CgroupInvalidParentPath())
+                }
+                _ => {}
+            }
```

Comment:
nit: could be
```suggestion
        if path.components().any(|c| c == Component::CurDir || c == Component::ParentDir) {
            return Err(Error::CgroupInvalidParentPath())
        }
```

Reply:
Changed it because we could also get rid of `parent_cgroup.is_absolute()` by checking the elements.